package.json

```diff
@@ -85,7 +85,7 @@
     "@signalk/instrumentpanel": "^0.3.0",
     "@signalk/maptracker": "^1.0.0",
     "@signalk/n2k-signalk": "^1.2.1",
-    "@signalk/nmea0183-signalk": "^2.2.0",
+    "@signalk/nmea0183-signalk": "^3.0.0-0",
```

Comment:
Should be 3.0.0, -0 is the prerelease version.

Reply:
:tired_face: Forgot to `add` before `commit --amend`